src/Fractal.php

```diff
@@ -26,7 +26,7 @@ public function __construct(Manager $manager)
 
     /**
      * @param null|mixed $data
-     * @param null|callable|\League\Fractal\TransformerAbstract $transformer
+     * @param null|string|callable|\League\Fractal\TransformerAbstract $transformer
```

Comment:
Parent method call `parent::create` also accepts the `$transformer` as an string.
I just updated the phpdoc to match the parent method phpdoc.